packages/gatsby-plugin-gatsby-cloud/src/components/Indicator.js

```diff
@@ -28,13 +30,29 @@ const PreviewIndicator = ({ children }) => (
   </>
 )
 
+const getContentSyncInfoFromURL = () => {
+  const urlSearchParams = new URLSearchParams(window.location.search)
+  const { mid: manifestId, plgn: pluginName } = JSON.parse(
+    atob(urlSearchParams.get(`csync`) || ``) || `{}`
+  )
+
+  if (!manifestId || !pluginName) {
+    return null
+  }
+
+  return { manifestId, pluginName }
+}
+
 let buildId = ``
 let pageData
 let latestCheckedBuild
 let refreshNeeded = false
 
 const Indicator = () => {
   const [buildInfo, setBuildInfo] = useState()
+  const [contentSyncInfo, setContentSyncInfo] = useState(null)
+  const usingContentSync = !!contentSyncInfo
```

Comment:
why is the `!!` needed here?

Reply:
It casts it to a boolean, the code would function without it but now TS marks usingContentSync as a boolean rather than as the type of contentSyncInfo